src/main/java/mate/jdbc/model/Driver.java

```diff
@@ -0,0 +1,70 @@
+package mate.jdbc.model;
+
+import java.util.Objects;
+
+public class Driver {
+    private Long id;
+    private String name;
+    private String licenseNumber;
+
+    public Driver() {
+    }
+
+    public Driver(Long id, String name, String licenseNumber) {
+        this.id = id;
```

Comment:
We don't have to set id it is automatically generated
```suggestion
    public Driver(String name, String licenseNumber) {
```